central/xlsx/device.go

```diff
@@ -0,0 +1,174 @@
+//
+// Copyright (C) 2023 IOTech Ltd
+//
+
+package xlsx
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/common"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/dtos"
+
+	"github.com/xuri/excelize/v2"
+)
+
+// deviceXlsx stores the worksheets processed result and the converted Device DTOs
+type deviceXlsx struct {
+	fieldMappings  map[string]mappingField // fieldMappings defines all the device fields with default values defined in the xlsx
+	Devices        []*dtos.Device          `json:"devices"`
+	ValidateErrors []error                 `json:"validateErrors,omitempty"`
+}
+
+// convertToDTO parses the Devices sheet and convert the rows to Device DTOs
+func (deviceXlsx *deviceXlsx) convertToDTO(xlsFile *excelize.File, protocol string) error {
+	var header []string
```

Comment:
in the beginning, implement and call a `validateDeviceSheetName` func to make sure all the necessary sheet names exist, or return a proper error

Reply:
Done